Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicStatisticsPage.java

```diff
@@ -0,0 +1,58 @@
+package edu.asu.spring.quadriga.domain.impl.workbench;
+
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+/**
+ * @description : Public Statistics Page class describing the properties of a
+ *              settings for Statisctics Page object
+ * 
+ * @author : Ajay Modi
+ * 
+ */
+public class PublicStatisticsPage implements IPublicPage {

```

Comment:
You do not need to do anything publicPages. That's just for the text blocks on the main page of the public page part. You need to create your own objects.